spec/acceptance/class_spec.rb

```diff
@@ -113,20 +113,20 @@ class { 'grafana':
     end
   end
 
-  context 'beta release' do
+  context 'update to beta release' do
     it 'works idempotently with no errors' do
       case fact('os.family')
       when 'Debian'
         pp = <<-EOS
         class { 'grafana':
-          version   => '6.0.0-beta3',
+          version   => 'latest',
           repo_name => 'beta',
         }
         EOS
       when 'RedHat'
         pp = <<-EOS
         class { 'grafana':
-          version       => '6.0.0',
+          version       => 'latest',
           rpm_iteration => 'beta3',
```

Comment:
Is this still needed if `latest` is used?

Reply:
What do you mean? The default for `grafana::version` is `"installed"`. So this test would not update to the latest beta version without setting it to something else.

Reply:
I've never used `rpm_iteration` but it looks to me that that's the way to specify the beta part while for deb it's specified in `version`. If you're switching to latest, I don't think it's needed to specify the `rpm_iteration`. Since I've never used it, I could be wrong.

Reply:
You definitely need to set both `grafana::version` - as already explained - and `grafana::repo_name`, so the beta `yum` repository is set up and the latest beta version can be found. I don't know whether `grafana::rpm_iteration` is mandatory, since I've never used `rpm_iteration` before either. Though you're probably right, that `grafana::version='latest'` and `rpm_iteration='beta3'` are potentially conflicting with each other (whenever there is no third beta iteration). So It probably is safer to drop it.

Reply:
Oh yes, `version => latest` makes sense.

Reply:
`rpm_iteration` will be dropped with the next push.